knowledge-challenge-app/src/Components/Header.js

```diff
@@ -5,7 +5,7 @@ import Dropdown from "react-bootstrap/Dropdown";
 export default function Header(props) {
   return (
     <div>
-      <div className="header-flex">
+      <div className={props.darkMode ? "header-flex-dark" : "header-flex"}>
```

Comment:
Great use of conditional rendering. I will definitely draw inspiration from this in future projects!